pkg/storage/metrics.go

```diff
@@ -155,6 +155,9 @@ var (
 	metaSysCount = metric.Metadata{
 		Name: "syscount",
 		Help: "Count of system KV pairs"}
+	metaWritesPerSecond = metric.Metadata{
+		Name: "writes_per_second",
```

Comment:
this should be a counter, rates need to be done by the adminUI/prometheus. It also needs a better name, the ones above are really not a good example to follow.